packages/rs-drive-abci/src/abci/handler/mod.rs

```diff
@@ -254,7 +255,7 @@ where
 
         let block_execution_outcome::v0::BlockExecutionOutcome {
             app_hash,
-            state_transition_results,
+            state_transitions_result,
```

Comment:
That's wrong. Rename

Reply:
GPT said it's a good and correct name for StateTransitionsProcessingResult structure